src/OmniSharp.Host/OmniSharpWorkspaceInitializer.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Composition.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OmniSharp.Options;
+using OmniSharp.Roslyn;
+using OmniSharp.Roslyn.Options;
+using OmniSharp.Services;
+
+namespace OmniSharp
+{
+    public class OmniSharpWorkspaceInitializer
+    {
+        private readonly OmniSharpWorkspace _workspace;
+        private readonly CompositionHost _compositionHost;
+        private readonly IConfiguration _configuration;
+        private readonly ILogger _logger;
+        private readonly IOptionsMonitor<OmniSharpOptions> _options;
+
+        public OmniSharpWorkspaceInitializer(
```

Comment:
This class is kind of weird. It looks like it can be reduced to a single static method.